### PR TITLE
[docs] - musescore/musescore/README.md - fixing tiny little issues with documentation (spelling, accent, et al.)

### PR DESCRIPTION
**Pull request by tmm2018**
[docs] - musescore/musescore/README.md - fixing tiny little issues with documentation (spelling, accent, et al.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ Kept as an example of how to integrate with a complex synthesizer.
 
 * **vtest** Visual tests. Compare reference images with current implementation
 
-* **thirdparty** Contains projects which are included for convenience, usually to integrate them into the build system to make them available for all supported platforms.
+* **third party** Contains projects which are included for convenience, usually to integrate them into the build system to make them available for all supported platforms.
```
Comment (Jojo-Schmitz) [inline README.md]: this is a folder name, without a space.

File: README.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ MuseScore is licensed under GPL version 2.0. See LICENSE.GPL in the same directo
 Disabled by default in the stable releases. See http://dev-list.musescore.org/Aeolus-Organ-Synth-td7578364.html
 Kept as an example of how to integrate with a complex synthesizer.
 
-* **assets** Graphical assets, use them if you need a MuseScore icon. For logo, color etc... see https://musescore.org/en/about/logos-and-graphics
+* **assets** Graphical assets, use them if you need a MuseScore icon. For the logo, color etc... see https://musescore.org/en/about/logos-and-graphics
```
Comment (Jojo-Schmitz) [inline README.md]: rather `for logos, colors`

**Comment by tmm2018** [comment]
all fixed.

**Comment by Jojo-Schmitz** [comment]
You'd need to squash these 3 commits into just one, and also sign the [CLA](https://musescore.org/cla)

**Comment by lasconic** [comment]
Thank you for the trouble but I'm happy with logo and color right now.

**Pull request closed without merging**